velox/exec/tests/RowContainerTest.cpp

```diff
@@ -1548,3 +1548,27 @@ TEST_F(RowContainerTest, toString) {
       rowContainer->toString(rows[2]),
       "{3, winter, 12, 123.00299835205078, null}");
 }
+
+TEST_F(RowContainerTest, initializeRowForReuseStringView) {
+  std::vector<TypePtr> keyTypes = {BIGINT(), VARCHAR()};
```

Comment:
nit: consider extending the test to cover all affected types: VARBINARY, ARRAY, MAP and ROW.

Reply:
Done.